sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarClient.swift

```diff
@@ -39,23 +40,41 @@ internal class RegistrarClient {
     private let registrationId: String
     /// URL Session.
     private let session: URLSession
-
+    /// Options provided to configure this RegistrarClient
+    internal let options: ClientOptions
+    
     // MARK: Initializers
 
     internal init(
         endpoint: String,
         credential: CommunicationTokenCredential,
         registrationId: String,
-        sessionConfiguration: URLSessionConfiguration? = nil
+        sessionConfiguration: URLSessionConfiguration? = nil,
+        authPolicy: Authenticating,
+        withOptions options: ClientOptions
+        
     ) throws {
-        guard let url = URL(string: endpoint) else {
+        guard let registrarURL = URL(string: endpoint) else {
             throw AzureError.client("Unable to form base registrar URL.")
         }
-
-        self.url = url
+        self.url = registrarURL
```

Comment:
nit: should be `registrarUrl`

Reply:
Thanks! Already fixed and will push the change.